dtfs-central-api/src/v1/controllers/tfm/amendments/tfm-get-amendments.controller.js

```diff
@@ -46,6 +46,10 @@ exports.getAllAmendmentsInProgress = async (req, res) => {
  * }]
  */
 const findAllAmendmentsByFacilityId = async (facilityId) => {
+  if (!ObjectId.isValid(facilityId)) {
```

Comment:
There is already some error handling in this file : Simply `cosole.error` and returning `null`. Where this is present, I've kept it. Where it isn't, we throw an error